common/scripts/kind_provisioner.sh

```diff
@@ -62,7 +62,7 @@ function load_cluster_topology() {
   export CLUSTER_SVC_SUBNETS
   export CLUSTER_NETWORK_ID
 
-  KUBE_CLUSTERS=$(jq '.[] | select(.kind == "Kubernetes" or .kind == null)' "${CLUSTER_TOPOLOGY_CONFIG_FILE}")
+  KUBE_CLUSTERS=$(jq '.clusters[] | select(.kind == "Kubernetes" or .kind == null)' "${CLUSTER_TOPOLOGY_CONFIG_FILE}")
```

Comment:
We won't want to merge this change here without some cross-repo changes as this change is in a common-files file which will get override and next make update-common.